1-js/13-modules/02-import-export/article.md

```diff
@@ -337,119 +338,124 @@ auth/
+
+## Ringkasan
 
-## Summary
+Berikut merupakan semua jenis `export` yang kita bahas di artikel ini dan sebelumnya.
 
-Here are all types of `export` that we covered in this and previous articles.
+Kamu dapat mengeceknya secara mandiri dengan membacanya dan mengingat apa maksudnya:
 
-You can check yourself by reading them and recalling what they mean:
+Ekspor:
 
-- Before declaration of a class/function/..:
-  - `export [default] class/function/variable ...`
-- Standalone export:
-  - `export {x [as y], ...}`.
-- Re-export:
-  - `export {x [as y], ...} from "module"`
-  - `export * from "module"` (doesn't re-export default).
-  - `export {default [as y]} from "module"` (re-export default).
+-   Sebelum deklarasi dari sebuah kelas/fungsi/..:
+    -   `export [default] class/function/variable ...`
+-   Ekspor mandiri:
+    -   `export { x [as y], ... } from "module"`
+-   Ekspor ulang:
+    -   `export {x [as y], ...} from "module"`
+    -   `export * from "module"` (doesn't re-export default).
+    -   `export {default [as y]} from "module"` (re-export default).
 
-Import:
+Impor:
 
-- Named exports from module:
-  - `import {x [as y], ...} from "module"`
-- Default export:  
-  - `import x from "module"`
-  - `import {default as x} from "module"`
-- Everything:
-  - `import * as obj from "module"`
-- Import the module (its code runs), but do not assign it to a variable:
-  - `import "module"`
+-   Ekspor bernama dari modul:
+    -   `import {x [as y], ...} from "module"`
+-   Ekspor bawaan:
+    -   `import x from "module"`
+    -   `import {default as x} from "module"`
+-   Semuanya:
+    -   `import * as obj from "module"`
+-   Impor modul (ini menjalankan kode), tetapi tidak disimpan kedalam variabel:
+    -   `import "module"`
 
-We can put `import/export` statements at the top or at the bottom of a script, that doesn't matter.
+Kita dapat meletakkan pernyataan `import/export` di bagian atas atau bawah dari skrip, itu tidak masalah.
+
+Jadi secara teknis kode ini tidak dipermasalahkan:
 
-So, technically this code is fine:
 ```js
 sayHi();
 
 // ...
 
-import {sayHi} from './say.js'; // import at the end of the file
+import { sayHi } from './say.js'; // impor dibagian bawah berkas
 ```
 
-In practice imports are usually at the start of the file, but that's only for more convenience.
+Dalam praktiknya, impor biasanya dilakukan pada awal sebuah berkas, tetapi itu hanya untuk memberikan kenyamanan lebih.
+
+**Harap diperhatikan bahwa pernyataan import/export tidak dapat digunakan jika didalam `{...}`.**
 
-**Please note that import/export statements don't work if inside `{...}`.**
+Sebuah impor bersyarat seperti ini tidak akan dapat berfungsi:
 
-A conditional import, like this, won't work:
 ```js
 if (something) {
-  import {sayHi} from "./say.js"; // Error: import must be at top level
+    import { sayHi } from './say.js'; // Galat: impor harus diluar pernnyataan if dann { ... }
```

Comment:
Error ini kaya hasil keluaran dari Javascriptnya gan ya? kalo keluaran Javascriptnya mending jangan diterjemahin, tapi dikasih catatan tambahan aja Error: import must be at top level (error harus diluar pernyataan if dan {...}/import didalam percabangan tidak akan bekerja). Ini errornya sama kaya _Error: 'import' and 'export' may only appear at the top level_. sama ada typo.. hehe

Reply:
okay mas, sudah saya perbaiki. ternyata banyak sekali typonya, maafkan mas. terima kasih karena review nya sangat detail